server/controller/index.js

```diff
@@ -57,3 +57,21 @@ exports.getId = async (ctx) => {
       return ctx.body;
     });
 };
+
+exports.markSpam = async (ctx) => {
+  const user = ctx.state.user.id;
+  if (!user) {
+    ctx.body = 'Not authorized';
+    return ctx.body;
+  }
+  const doc = await Question.findOne({ _id: ctx.params.id })
+    .then((data) => data )
```

Comment:
You can remove the parentheses around data. ESLint allows `.then(data => data)` apologies for the mistake.